src/dir.targets

```diff
@@ -43,9 +43,21 @@
     <!-- Targets from buildtools that restore packages before building. -->
   <!-- TODO: ideally replaced by a standard "dotnet restore" over the solution. -->
   <Import Condition="Exists('$(ToolsDir)\packageresolve.targets')" Project="$(ToolsDir)\packageresolve.targets" />
+
+  <ItemGroup Condition="'$(NetCoreBuild)' == 'true' and '$(UpdateNuGetPackageDependencies)' == 'true'">
+    <NuSpecFile Include="$(ProjectDir)build\NuGetPackages\$(MSBuildProjectName).nuspec" Condition="Exists('$(ProjectDir)build\NuGetPackages\$(MSBuildProjectName).nuspec')">
+      <Visible>false</Visible>
+    </NuSpecFile>
+    <NuSpecFile Include="$(ProjectDir)build\NuGetPackages\$(MSBuildProjectName).Core.nuspec" Condition="Exists('$(ProjectDir)build\NuGetPackages\$(MSBuildProjectName).Core.nuspec')">
```

Comment:
This sucks but our `.csproj`s don't contain ".Core" like `Microsoft.Build.Tasks.Core` so I had to add a fallback here 😢 

Solutions:
1. Rename `Microsoft.Build.Tasks.csproj` to `Microsoft.Build.Tasks.Core.csproj`
2. Rename `Microsoft.Build.Tasks.Core.nuspec` to `Microsoft.Build.Tasks.nuspec`
3. Leave this hack...